lib/libutee/include/pta_invoke_tests.h

```diff
@@ -73,5 +73,23 @@
  */
 #define PTA_INVOKE_TESTS_CMD_LOCKDEP		8
 
+#define PTA_INVOKE_TESTS_MODE_ECB		0
+#define PTA_INVOKE_TESTS_MODE_CBC		1
+#define PTA_INVOKE_TESTS_MODE_CTR		2
+#define PTA_INVOKE_TESTS_MODE_XTS		3
+#define PTA_INVOKE_TESTS_MODE_GCM		4
+
+/*
+ * AES performance tests
+ *
+ * [in]     value[0].a	Top 16 bits Decrypt, low 16 bits key size in bytes
+ * [in]     value[0].b	Mode, one of TEE_MODE_{ECB_NOPAD,CBC_NOPAD,CTR,XTS,GCM}
+ * [in]     value[1].a	repetition count
+ * [in]     value[1].b	unit size
+ * [in]     memref[2]	In buffer
+ * [in]     memref[3]	Out buffer
+ */
+#define PTA_INVOKE_TEST_CMD_AES_PERF_CMD	9
```

Comment:
Why the `_CMD` suffix? I suggest `PTA_INVOKE_TEST_CMD_AES_PERF`.

Reply:
I'll fix